app/src/main/java/org/android/go/sopt/presentation/auth/LoginActivity.kt

```diff
@@ -70,20 +77,30 @@ class LoginActivity : AppCompatActivity() {
         return binding.etId.text.isNotEmpty() && binding.etPw.text.isNotEmpty()
     }
 
-    private fun setLoginData() {
-        if (isCorrect()) {
-            showToast(getString(R.string.sign_in_done))
-            val loginIntent = Intent(this, ProfileActivity::class.java)
-            loginIntent.putExtra(IntentKey.USER, user)
-            startActivity(loginIntent)
-            finish()
-        } else {
-            showSnackBar(getString(R.string.login_incorrect), binding)
-        }
-    }
-
-    private fun isCorrect(): Boolean {
-        return binding.etId.text.toString() == user?.id && binding.etPw.text.toString() == user?.pw
+    private fun completeLogin() {
+        loginService.login(
+            with(binding) {
+                RequestLoginDto(
+                    etId.text.toString(),
+                    etPw.text.toString()
+                )
+            }
+        ).enqueueUtil(
+            onSuccess = {
+                startActivity(
+                    Intent(this@LoginActivity, MainActivity::class.java)
+                )
+                showToast(R.string.sign_in_done.toString())
```

Comment:
```suggestion
                showToast(getString(R.string.sign_in_done))
```
혹시 의도가 이게 맞았을까요 ? 기존 코드는 Int 값을 toString() 때려버리는 것 같아서 의아하네요